qcodes/instrument_drivers/Keysight/private/Keysight_344xxA_submodules.py

```diff
@@ -172,12 +173,17 @@ def __init__(self, parent: '_Keysight_344xxA', name: str, **kwargs):
             option) or 2,000,000 readings (with the MEM option)"""))
 
         if self.parent.has_DIG:
+            if self.parent.has_MEM:
+                _max_pretrig_count = 2e6
```

Comment:
this should really be int(2e6) as 2e6 is a float

Reply:
done

Reply:
Correct. And the _max_pretrig_count below should be int(5e4)